tests/hotplug/cpu.go

```diff
@@ -209,20 +240,10 @@ var _ = Describe("[sig-compute][Serial]CPU Hotplug", decorators.SigCompute, deco
 			migration, err = virtClient.VirtualMachineInstanceMigration(vm.Namespace).Create(migration, &metav1.CreateOptions{})
 			Expect(err).ToNot(HaveOccurred())
 
-			// This validation is being added here because currently it is very
-			// costly to run it in a separate test.
-			// Chaning the WorkloadUpdateMethods takes a very long time and
-			// restarts the whole control plane
-			By("Making sure that a hot CPU change is not allowed during an inflight migration")
-			patchData, err := patch.GenerateTestReplacePatch("/spec/template/spec/domain/cpu/sockets", 1, 2)
-			Expect(err).NotTo(HaveOccurred())
-			_, err = virtClient.VirtualMachine(vm.Namespace).Patch(context.Background(), vm.Name, types.JSONPatchType, patchData, &k8smetav1.PatchOptions{})
-			Expect(err).To(MatchError(ContainSubstring("cannot update while VMI migration is in progress")))
-
```

Comment:
How do we prevent the issues that were protected by this?

Reply:
That now happens at the VM controller level, instead of the webhook, which I'm not sure how to test reliably...

Reply:
Can you point me to the code?

Reply:
https://github.com/kubevirt/kubevirt/pull/11054/files#diff-3f56578b7747051a1e3dd5eb83708d7a4fff2d205ac710dcb0fdf127f1e3afe5R651-R658